src/main/webapp/script.js

```diff
@@ -281,3 +307,14 @@ function setImageFromBlobstore(imageBlobKey, imageId) {
     document.getElementById(imageId).src = URL.createObjectURL(blobContent);
   });
 }
+
+//Function that creates an image element that corresponds to the provided blobkey and returns it
+function createImageFromBlobstore(imageBlobKey) {
```

Comment:
Hmm this function will create and return an `img` element *without* an `src` attribute, then populate it later on once the fetch complete. That feels a bit awkward, but probably fine from a functional perspective.

Is there a reason the servlet doesn't just return the image URL? Since it doesn't look like the client can do much with the `blobKey` alone.

Reply:
I'm pretty sure Isabelle had an issue in the walkthroughs with returning the url, not completely sure though. I think in all the tutorials the servlet returns the blobkey? Gotta check with Jess on this.

Reply:
Okay, not anything that needs to be addressed in this CL, was just curious!